skbio/stats/distance/_regression.py

```diff
@@ -0,0 +1,170 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+
+import six
+import numpy as np
+from skbio.util._decorator import experimental
+from skbio.util._misc import check_random_state
+from skbio.stats.distance._mantel import _order_dms
+
+from skbio.stats.distance import DistanceMatrix
+
+@experimental(as_of="0.4.0")
+def linregress(y, *args, **kwargs):
```

Comment:
> But perhaps that name is misleading. Let me change it to mrm.

:+1: for `mrm`. The MRM page I linked to above indicates the regression doesn't necessarily have to be linear. Is that the case for this function? The docstring indicates linear regression.

> In the meantime, @JWDebelius talked about stripping out the MLR portion and making it its own functionality. While I think this is a good idea, I'm not sure where this MLR should belong.
I could put it in a new folder skbio.stats.regression.py. Any thoughts?

MLR appears to be implemented in [statsmodels](http://www.datarobot.com/blog/multiple-regression-using-statsmodels/), [scikit-learn](http://stackoverflow.com/a/11479279/3776794), and [scipy](http://stackoverflow.com/questions/17679140/multiple-linear-regression-with-python). What about using one of these implementations instead?